pkg/sdk/privileges.go

```diff
@@ -71,6 +72,9 @@ const (
 	AccountObjectPrivilegeMonitor            AccountObjectPrivilege = "MONITOR"
 	AccountObjectPrivilegeUsage              AccountObjectPrivilege = "USAGE"
 
+	// -- For EXTERNAL VOLUME
+	// AccountObjectPrivilegeUsage              AccountObjectPrivilege = "USAGE" (duplicate)
```

Comment:
why this is commented out?

Reply:
So it seems like we're adding the privileges as they appear in the documentation, e.g. for External Volume ... For Tables ... etc. For some of them, they can duplicate (e.g. tables and iceberg tables may have "SELECT") and the practice in this file was to comment the duplicate with the comment `(duplicate)` and the end.

Reply:
Oh, okay. We should deal with that. Maybe as a subtask in redesign grants? (close after reading and adding it there if you agree)